integration-tests/docker-compose.yaml

```diff
@@ -55,14 +55,7 @@ services:
     depends_on:
       - db
     environment:
-      - BOB_DB_HOST=db
-      - BOB_DB_PORT=5432
-      - BOB_DB_USER=bob
-      - BOB_DB_NAME=bob
-      - BOB_CONNECT_TIMEOUT=0
-      - BOB_READ_TIMEOUT=0
-      - BOB_WRITE_TIMEOUT=0
-      - BOB_CALL_TIMEOUT=0
+      - BOB_POSTGRES_HOST=db
```

Comment:
lets keep the timeout values here, to the defaults maybe. will protect against the issues we faced.

Reply:
oh no! I had to start the integration tests four times to get them green and now you want changes:)

Reply:
so I switch them back to 1000/30000/30000/40000?

Reply:
yes